pycbc/results/color.py

```diff
@@ -10,6 +10,19 @@
     'V1': '#9b59b6',  # magenta/purple
 }
 
+_source_color_map = {
```

Comment:
I took the color scheme from this classification chart: https://emfollow.docs.ligo.org/userguide/_images/content-1.svg

Reply:
Sounds good to me.